modules/openy_gc_auth/modules/openy_gc_auth_reclique/openy_gc_auth_reclique.services.yml

```diff
@@ -6,4 +6,4 @@ services:
       - { name: 'event_subscriber' }
   openy_gc_auth_reclique_client:
     class: '\Drupal\openy_gc_auth_reclique\RecliqueClientService'
-    arguments: ['@config.factory', '@http_client']
+    arguments: ['@logger.factory','@config.factory', '@http_client']
```

Comment:
Suuuuper nitpicky, but there should be a space after the comma here. See https://github.com/fivejars/openy_gated_content/pull/192/commits/cdb685824095fa5ec6c968387c213f5f6bf31199

Reply:
Good catch - fixed